### ISSUE TITLE
Italian translation

### ISSUE THREAD

**Issue by gianluca-salogni** [LAION-AI/Open-Assistant#1347]
"Non appopiato" button label, should be changed in "Non appropriato"

**Comment by solyarisoftware** [comment]
Anyway I want to really thank you the guy submitted the first Italian translation @Jac-Zac !
https://github.com/LAION-AI/Open-Assistant/pull/1244

**Comment by gianluca-salogni** [comment]
Molto gentile
Grazie
Gianluca

Il mer 8 feb 2023, 09:56 Giorgio Robino ***@***.***> ha
scritto:

**Comment by alek3y** [comment]
Also, I would like to add that "Non è una buona risposta alla richiesta?" makes the "Si"/"No" option a bit confusing. One could say "No" as to imply "No, non è una buona risposta alla richiesta".

**Comment by gianluca-salogni** [comment]
Indeed it is. Could be "è una cattiva risposta alla richiesta?" or in a more formal way "è una risposta non pertinente alla richiesta?"

**Comment by alek3y** [comment]
I think the negation here is the problem, which might be why they avoided it in english.

**Comment by Jac-Zac** [comment]
What would you prefer for me to write since I already have a pull request I can change it maybe

**Comment by Jac-Zac** [comment]
To me: "è una risposta non pertinente alla richiesta?" sounds the best thus I think I'll go with that for now. Unless someone is against it

**Comment by alek3y** [comment]
What about "È una risposta inadatta alla richiesta?" perhaps? Just as a suggestion.

**Comment by Jac-Zac** [comment]
Even thought I agree it perhaps is slightly better

**Comment by alek3y** [comment]
I feel like it has to be as clear as possible to avoid confusion, given they might discard the replies with a lot of "Si".

**Comment by Jac-Zac** [comment]
Now I fill like the bigger problem is in the: [account page](https://open-assistant.io/it/account) because it doesn't take account for translations in some places. Firstly because some fields should be added to every translation. And also some code should be changed inside the index.tsx

**Comment by Jac-Zac** [comment]
Also because the account page uses: ```import { LeaderboardEntity, LeaderboardTimeFrame } from "src/types/Leaderboard";``` and the leaderboard file doesn't use translation when writing: "day", etc ... Also some terms should be added for every translation such as: "Your statistics" and other. I could do it for the Italian translation but for the other one since I do not know the lagueges. Thus I'd like to get some feedback to know how they'd like to handle this ? Maybe I could add it to the italian translation and then people will add it to the other translation in the future. (But this is not something of extream ungercy since it is not something that creates wrong traning data or makes the app unusable)

**Comment by bitplane** [comment]
>  Also some terms should be added for every translation such as: "Your statistics" and other.

Yes all text displayed to users should be replaced in this way. If there are hard coded English strings in there then it's a bug for sure
